deploy/runtools/runtime_config.py

```diff
@@ -343,6 +466,9 @@ def deep_merge(a: dict, b: dict) -> dict:
             self.synthprint_config.start = runtime_dict['synth_print'].get("start", "0")
             self.synthprint_config.end = runtime_dict['synth_print'].get("end", "-1")
             self.synthprint_config.cycle_prefix = runtime_dict['synth_print'].get("cycle_prefix", "yes") == "yes"
+        self.default_plusarg_passthrough = ""
+        if 'plusarg_passthrough' in runtime_dict['target_config']:
```

Comment:
Similar comment to one above about defaults.

Reply:
fixed